pkl-core/src/main/java/org/pkl/core/JsonRenderer.java

```diff
@@ -102,6 +102,12 @@ public void visitDataSize(DataSize value) {
           String.format("Values of type `DataSize` cannot be rendered as JSON. Value: %s", value));
     }
 
+    @Override
+    public void visitBytes(byte[] value) {
+      throw new RendererException(
+          String.format("Values of type `Bytes` cannot be rendered as JSON. Value: %s", value));
```

Comment:
I wonder if it makes sense to suggest defining a converter in these error messages directly to make them more actionable.